packages/accounts-controller/package.json

```diff
@@ -42,7 +42,7 @@
   },
   "devDependencies": {
     "@metamask/auto-changelog": "^3.4.3",
-    "@metamask/keyring-controller": "^8.1.0",
+    "@metamask/keyring-controller": "^9.0.0",
```

Comment:
The linter is failing because we need to bump the peerDependency as well

Reply:
We'll want to release the accounts controller in this release as well, so that it's compatible with the new keyring controller release.

It'll be a breaking change because this peer dependency update is a breaking change

Reply:
Fixed: https://github.com/MetaMask/core/pull/2052/commits/a2b171d75a12a2cc71f59d8cf8d1d164df7ad657